packages/api-specification/generate-docs.js

```diff
@@ -0,0 +1,224 @@
+const fs = require('fs');
+
+const output = JSON.parse(fs.readFileSync('type-info.json'));
+const testMatrixJson = JSON.parse(fs.readFileSync('test-report.json'));
+
+let mdObject = {};
+
+// Read as 10% or less, 25% or less etc
+// Colors are the colors for the badges on http://shields.io
+const badgeColors = {
+  10: 'red',
+  25: 'yellow',
+  50: 'yellowgreen',
+  75: 'green',
+  100: 'brightgreen'
+};
+
+const getBadgeColor = (percent) => {
+  let colorCode = '';
+  Object.keys(badgeColors).some((key) => {
+    if (percent <= parseInt(key)) {
+      colorCode = badgeColors[key];
+      return true;
+    }
+    return false;
+  });
+  return colorCode;
+};
+
+// === Markdown Generation Methods ===
+const generateModule = (moduleObj) => {
+  if (moduleObj.children) {
+    moduleObj.children.forEach((child) => {
+      render(child);
+    });
+  }
+};
+
+const generateClass = (classObj) => {
+  mdObject[classObj.name] = `# ${classObj.name}  \n`;
+  if (classObj.children) {
+    classObj.children.forEach((child) => {
+      render(child, classObj.name);
+    });
+  }
+};
+
+const generateMethod = (methodObj, className) => {
+  let methodData = '';
```

Comment:
You could just define this as `const methodData = methodObj.flags.isStatic ? ' (static)' : '';`